src/sentry/api/endpoints/project_stacktrace.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.http import safe_urlopen
+
+
+class ProjectStacktraceEndpoint(ProjectEndpoint):
+    def get(self, request: Request, project, transaction_id) -> Response:
+        if not features.has("organizations:profiling", project.organization, actor=request.user):
+            return Response(status=404)
```

Comment:
It'll be nice to see a basic test checking that this returns 404 if `organizations:profiling` is absent from the features set.

Reply:
Yep looks good.